stock_request_purchase/models/purchase_order_line.py

```diff
@@ -12,6 +12,15 @@ class PurchaseOrderLine(models.Model):
         comodel_name="stock.request", string="Stock Requests", copy=False
     )
 
+    def unlink(self):
```

Comment:
Include docstring about the general what, the whys, and so on.

Reply:
done